### ISSUE TITLE
Stripe statement descriptor

### ISSUE THREAD

**Issue by efavaro** [wagnerwagner/merx#21]
After a purchase I noticed in the support phone number set in Stripe appears in the bank statement descriptor.
I tried adding the shortened descriptor in Stripe config, but nothing changes and the number is still there. 

While reading this https://stripe.com/docs/statement-descriptors I was wondering that this probably can be achieved by adding it to here https://github.com/wagnerwagner/merx/blob/7a52a1693d26478375a87238a093b9976fe2cb86/src/payment.php#L91 using an option perhaps?

**Comment by tobiasfabian** [comment]
In the [develop-branch](https://github.com/wagnerwagner/merx/tree/develop) I’ve updated the [Cart::getStripePaymentIntent()](https://github.com/wagnerwagner/merx/blob/86f516f30d764ac219517d66b13501547cd88fda/src/cart.php#L130) method. You can now pass additional `$params`.

The following should work (untested)

```
$paymentIntent = $cart->getStripePaymentIntent([
  'statement_descriptor' => 'Custom descriptor',
]);
```

Further information:  
- https://stripe.com/docs/payments/payment-intents#dynamic-statement-descriptor
- https://stripe.com/docs/api/payment_intents/create?lang=php

**Comment by efavaro** [comment]
Thank you! I will try it out.

**Comment by tobiasfabian** [comment]
Published in [1.5.0-rc.1](https://github.com/wagnerwagner/merx/releases/tag/1.5.0-rc.1)
